pkg/daemon/ceph/client/radosnamespace.go

```diff
@@ -0,0 +1,105 @@
+/*
+Copyright 2022 The Rook Authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package client
+
+import (
+	"encoding/json"
+	"syscall"
+
+	"github.com/pkg/errors"
+	"github.com/rook/rook/pkg/clusterd"
+	"github.com/rook/rook/pkg/util/exec"
+)
+
+// CreateRadosNamespace create a rados namespace in a pool.
+// poolName is the name of the ceph block pool, the same as the CephBlockPool CR name.
+func CreateRadosNamespace(context *clusterd.Context, clusterInfo *ClusterInfo, poolName, namespaceName string) error {
+	logger.Infof("creating rados namespace %q", namespaceName)
```

Comment:
The rook operator can operate on multiple resources in multiple namespaces. All info/debug messages must list the namespaced name so context is clear.

On which pool? For which cephcluster?